[Bug] Run python3 -m sglang.check_env reported an error in NPU

### Checklist

- [x] I searched related issues but found no solution.
- [x] The bug persists in the latest version.
- [x] Issues without environment info and a minimal reproducible demo are hard to resolve and may receive no feedback.
- [x] If this is not a bug report but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Describe the bug

Using Docker Images "main-cann8.5.0-910b", run `python3 -m sglang.check_env` failed, but using old image "v0.5.7-cann8.3.rc2-910b" succeeded



### Reproduction

`python3 -m sglang.check_env`

### Environment

main-cann8.5.0-910b
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/python3.11.14/lib/python3.11/site-packages/sglang/check_env.py", line 592, in <module>
    env.check_env()
  File "/usr/local/python3.11.14/lib/python3.11/site-packages/sglang/check_env.py", line 133, in check_env
    env_info.update(self.get_info())
                    ^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/site-packages/sglang/check_env.py", line 317, in get_info
    cuda_info.update(self._get_cann_version_info())
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/site-packages/sglang/check_env.py", line 349, in _get_cann_version_info
    npu_info.update(self._get_cann_info(CANN_HOME))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/site-packages/sglang/check_env.py", line 367, in _get_cann_info
    subprocess.check_output([bisheng, "--version"]).decode("utf-8").strip()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/subprocess.py", line 548, in run
    with Popen(*popenargs, **kwargs) as process:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/python3.11.14/lib/python3.11/subprocess.py", line 1026, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/python3.11.14/lib/python3.11/subprocess.py", line 1955, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/Ascend/cann-8.5.0/compiler/ccec_compiler/bin/bisheng'
[ERROR] 2026-04-15-03:13:07 (PID:541487, Device:-1, RankID:-1) ERR99999 UNKNOWN applicaiton exception
```



v0.5.7-cann8.3.rc2-910b
```
Python: 3.11.13 (main, Nov 20 2025, 16:02:27) [GCC 11.4.0]
NPU available: True
NPU 0,1,2,3,4,5,6,7: Ascend910B4
CANN_HOME: /usr/local/Ascend/ascend-toolkit/latest
CANN: 8.3.0.2.220:8.3.RC2
BiSheng: 2025-10-24T18:53:37+08:00 clang version 15.0.5 (clang-5c68a1cb1231 flang-5c68a1cb1231)
Ascend Driver Version: 25.5.0
PyTorch: 2.8.0+cpu
sglang: 0.5.7
sgl_kernel: Module Not Found
flashinfer_python: Module Not Found
flashinfer_cubin: Module Not Found
flashinfer_jit_cache: Module Not Found
triton: Module Not Found
transformers: 4.57.1
torchao: 0.9.0
numpy: 1.26.4
aiohttp: 3.13.2
fastapi: 0.128.0
hf_transfer: 0.1.9
huggingface_hub: 0.36.0
interegular: 0.3.3
modelscope: 1.33.0
orjson: 3.11.5
outlines: 0.1.11
packaging: 25.0
psutil: 7.1.3
pydantic: 2.12.5
python-multipart: 0.0.21
pyzmq: 27.1.0
uvicorn: 0.40.0
uvloop: 0.22.1
vllm: Module Not Found
xgrammar: 0.1.27
openai: 2.6.1
tiktoken: 0.12.0
anthropic: 0.75.0
litellm: Module Not Found
decord2: 3.0.0
torch_npu: 2.8.0.post2.dev20251113
sgl-kernel-npu: 0.1.0
deep_ep: 1.0.0+3f61eab6.cann.8.3.rc2.b210
Ascend Topology:
           NPU0       NPU1       NPU2       NPU3       NPU4       NPU5       NPU6       NPU7       CPU Affinity
NPU0       X          HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       144-167
NPU1       HCCS       X          HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       144-167
NPU2       HCCS       HCCS       X          HCCS       HCCS       HCCS       HCCS       HCCS       96-119
NPU3       HCCS       HCCS       HCCS       X          HCCS       HCCS       HCCS       HCCS       96-119
NPU4       HCCS       HCCS       HCCS       HCCS       X          HCCS       HCCS       HCCS       0-23
NPU5       HCCS       HCCS       HCCS       HCCS       HCCS       X          HCCS       HCCS       0-23
NPU6       HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       X          HCCS       48-71
NPU7       HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       HCCS       X          48-71

Legend:

  X    = Self
  SYS  = Path traversing PCIe and NUMA nodes. Nodes are connected through SMP, such as QPI, UPI.
  PHB  = Path traversing PCIe and the PCIe host bridge of a CPU.
  PIX  = Path traversing a single PCIe switch
  PXB  = Path traversing multipul PCIe switches
  HCCS = Connection traversing HCCS.
  NA   = Unknown relationship.

ulimit soft: 1048576

```

--- a/python/sglang/check_env.py
+++ b/python/sglang/check_env.py
@@ -362,7 +362,7 @@ class NPUEnv(BaseEnv):
         else:
             cann_info["CANN"] = "Not Available"
         try:
-            bisheng = os.path.join(CANN_HOME, "compiler/ccec_compiler/bin/bisheng")
+            bisheng = os.path.join(CANN_HOME, "tools/bisheng_compiler/bin/bisheng")
             bisheng_output = (
                 subprocess.check_output([bisheng, "--version"]).decode("utf-8").strip()
             )

--- a/python/sglang/check_env.py
+++ b/python/sglang/check_env.py
@@ -363,6 +363,9 @@ class NPUEnv(BaseEnv):
             cann_info["CANN"] = "Not Available"
         try:
             bisheng = os.path.join(CANN_HOME, "tools/bisheng_compiler/bin/bisheng")
+            if not os.path.isfile(bisheng)
+                # Check path for old CANN version
+                bisheng = os.path.join(CANN_HOME, "compiler/ccec_compiler/bin/bisheng")
             bisheng_output = (
                 subprocess.check_output([bisheng, "--version"]).decode("utf-8").strip()
             )

--- a/python/sglang/check_env.py
+++ b/python/sglang/check_env.py
@@ -363,7 +363,7 @@ class NPUEnv(BaseEnv):
             cann_info["CANN"] = "Not Available"
         try:
             bisheng = os.path.join(CANN_HOME, "tools/bisheng_compiler/bin/bisheng")
-            if not os.path.isfile(bisheng)
+            if not os.path.isfile(bisheng):
                 # Check path for old CANN version
                 bisheng = os.path.join(CANN_HOME, "compiler/ccec_compiler/bin/bisheng")
             bisheng_output = (

